pkg/clusteragent/admission/mutate/autoinstrumentation/namespace_mutator.go

```diff
@@ -205,14 +226,8 @@ func (m *mutatorCore) injectTracers(pod *corev1.Pod, config extractedPodLibInfo)
 	return lastError
 }
 
-func (m *mutatorCore) newContainerMutators(requirements corev1.ResourceRequirements) containerMutators {
-	return containerMutators{
-		containerResourceRequirements{requirements},
-		containerSecurityContext{m.config.initSecurityContext},
-	}
-}
-
-func (m *mutatorCore) newInjector(startTime time.Time, pod *corev1.Pod, opts ...injectorOption) podMutator {
+func (m *mutatorCore) newInjector(startTime time.Time, pod *corev1.Pod, libOpts libRequirementOptions) podMutator {
```

Comment:
Did some work to remove the indirection here so that when we set up the injector there's _less stuff_.